changes/1307.bugfix.md

```diff
@@ -1 +0,0 @@
-Wrong typehint for `InviteGuild.features`.
```

Comment:
Please dont delete this one

Reply:
Fixed, I was confused about how this changes/ stuff was working